backend/infrahub/git/tasks.py

```diff
@@ -385,10 +468,16 @@ async def generate_request_artifact_definition(
             context=context,
         )
 
-        await get_workflow().submit_workflow(
-            workflow=REQUEST_ARTIFACT_GENERATE, context=context, parameters={"model": request_artifact_generate_model}
+        batch.add(
```

Comment:
is this batch really required ? it will create them faster in Prefect but it won't really speed up the execution

Reply:
you are right, it's similar to the discussion here: https://github.com/opsmill/infrahub/pull/6393#issuecomment-3193535718

it just allows shifting the bottleneck on the Prefect API servers (and then scale them out if necessary to achieve better performance)